app/views/password_resets/edit.html.haml

```diff
@@ -0,0 +1,14 @@
+%section.reset_password.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      =form_tag('/reset_password') do
```

Comment:
I had a bit of a problem here, because the form kept posting to `/reset_password/:token`. I tried around with different path helpers and routing, but none of them seemed to just give me `/reset_password` by itself...I might try again.